### ISSUE TITLE
[BUG] Dispatcher Module with Clusterer - no cid in branch added for pings

### ISSUE THREAD

**Issue by digipigeon** [OpenSIPS/opensips#1783]
**OpenSIPS version you are running**

```
version: opensips 3.0.0-beta (x86_64/linux)
flags: STATS: On, DISABLE_NAGLE, USE_MCAST, SHM_MMAP, PKG_MALLOC, Q_MALLOC, F_MALLOC, HP_MALLOC, DBG_MALLOC, CC_O0, FAST_LOCK-ADAPTIVE_WAIT
ADAPTIVE_WAIT_LOOPS=1024, MAX_RECV_BUFFER_SIZE 262144, MAX_LISTEN 16, MAX_URI_SIZE 1024, BUF_SIZE 65535
poll method support: poll, epoll, sigio_rt, select.
git revision: 9b47639e3
main.c compiled on  with gcc 7
```

**Describe the bug**

Running OpenSIPs with the Dispatcher module and Clusterer module.
When the Dispatcher module sends OPTIONS pings it fails to include the `cid=` tag which would be used to re-route reply to the sending OpenSIPs instance.

**To Reproduce**

Run OpenSIPs with Dispatcher and Clusterer Module (Anycast Environment).
Observe outbound OPTIONS messages, specifically the `via` header, note the lack of `cid=`

I am not using `cluster_sharing_tag`. The behavior I am trying to achieve would is per zone/server state self-determination (This way we can determine if latency becomes too high to a particular destination (from a particular server/zone), we can mark it as inactive, opposed to sharing a state between all dispatchers. Also means that `"dispatcher", "cluster_id"` should not be set (however I have tried with as well).

**Expected behavior**

Although the outbound OPTIONS messages may not go through the tm module. We should expect it to be signed with the same `cid` tag to allow correct re-routing of the reply back to the sending.

**Comment by digipigeon** [comment]
This hack appears to fix the problem:
```
local_route {
	if (is_method("OPTIONS")) {
		replace("branch=", "cid={{server_id}};branch=");
	}
}
```

**Comment by bogdan-iancu** [comment]
@digipigeon , I think you forgot to set the [tm_replication_cluster](https://opensips.org/html/docs/modules/3.0.x/tm.html#param_tm_replication_cluster) param in the tm module.

**Comment by digipigeon** [comment]
@bogdan-iancu I doubt that this is the case based on how I remember my tests going.

`tm_replication_cluster` was set if the packet originated externally to opensips it was appended correctly. However if OpenSIPs generated the ping message itself it did not append the `cid`.

**Comment by bogdan-iancu** [comment]
OK, your revision is bit old (19th of May) - the issue you have was fixed on 3rd of Jun ( see 8c9704f6c3add396e478658d0c15b1c818ec8ea7). Pull the latest package or code for 3.0 and try it again.

**Comment by digipigeon** [comment]
This has fixed it.
